docker/package/tezos_voting_wizard.py

```diff
@@ -50,14 +50,15 @@
 welcome_text = """Tezos Voting Wizard
 
 Welcome, this wizard will help you to vote in the Tezos protocol amendment process.
-Please note that for this you need to already have set up the baking infrastructure,
-normally on mainnet, as only bakers can submit ballots or proposals.
+Please note that to vote on mainnet, you need to already have one of the following setups:
+- the baking infrastructure on mainnet;
+- a connected ledger device with Tezos Wallet app running;
 
 If you have installed tezos-baking, you can run Tezos Setup Wizard to set it up:
 tezos-setup-wizard
 
 Alternatively, you can use this wizard for voting on a custom chain,
-which also needs to be set up already.
+baking on which needs to be set up already.
 
 All commands within the service are run under the 'tezos' user.
```

Comment:
I think that we should reconsider this part, because it gives a lot of info that in most cases people won't be interested in.

IMO here we should only mention the _actual_ minimum requirements for voting.

I think that we can avoid mentioning custom networks too, and leave that to be explained in the docs instead.

As for the baking setup(and how to run it), I suggest mentioning it later, in case it isn't found.

Reply:
That's actually a tricky part, because the user may provide _various_ `tezos-client` data directories that may potentially belong to a user that is different from the one that started `tezos-voting-wizard`. So we should either dynamically define which user is going to run `tezos-client` commands or say that they always run under the current user and if one needs the other user, he has to do `sudo -u tezos-voting-wizard`

Reply:
Also, if the target is to have a wizard for voting with minimum possible requirements, I think it should be moved to a dedicated package, because currently `tezos-baking` has quite a lot of redundant dependencies for voting

Reply:
I agree about the dedicated package, though I think in this case especially #388 should be solved before that or at the same time. It might be best to open another issue for it instead of separating it here.

I'm not sure running under the current user/through `sudo -u` will be good enough though :thinking: I can imagine there can be dependency on the two users (starting and e.g. `tezos`) being different and having different rights, but even if not, using e.g. `sudo -u tezos tezos-voting-wizard` looks unnecessarily cluttered to me. I think it's best to indeed define the second user dynamically. Looks like we only call `tezos-client` after collecting all the config data anyway, so it shouldn't be a problem to find out the user to whom `client_data_dir` belongs. Or can you think of any problems with that approach?

Reply:
IDK about moving the wizard somewhere else :shrug: 

IMO it can stay in the same package for now, because as I mentioned even tho we want to be able to support minimal requirements it wouldn't hurt to give a better experience for those that have set up baking already with the other wizard.

There still seems to be quite a bit of overlap between the two IMO

Reply:
> I think it's best to indeed define the second user dynamically. Looks like we only call `tezos-client` after collecting all the config data anyway, so it shouldn't be a problem to find out the user to whom `client_data_dir` belongs. Or can you think of any problems with that approach?

I don't think that it should cause any problems. And I'm in favor of implementing it this way

Reply:
> For both (2) and (3) I think that this wizard should guide into importing the key of the voter, rather than juggling with tezos-client data dirs. In the worst case scenario one would have to go through the import procedure on a machine where they already performed it (in a different dir)

Do you propose to use a dedicated data directory for each voting wizard invocation?

Reply:
In general, it shouldn't be too inconvenient. Because one only has to import a key in order to vote (it isn't required to set up a ledger to bake and register the key as a delegate). As for different directories, I guess we can have `tezos-client` data directories per network (e.g. marked with `chain_id` obtained from the corresponding node). This way we'll avoid reimporting the key next time the user will vote because he'll highly likely use the same key.

Reply:
> Do you propose to use a dedicated data directory for each voting wizard invocation?

Ah no, perhaps that was confusing. What I was saying is that in the remote case that one has already that key imported 
 somewhere (e.g. they did so using `tezos-client` with their own user and the default config) then they'd have to re-import it with what I propose, which is IMO not that inconvenient (and not happening in most cases).

What I was suggesting is to use the `tezos` user and the `--network` argument to determine the data dir, like the setup wizard does (with `get_data_dir` AFAIK).